src/components/common/TransactionRow.js

```diff
@@ -132,55 +134,80 @@ export class TransactionRowComponent extends React.PureComponent<Props> {
-            <View style={styles.iconContainer}>{thumbnailPath ? <Image style={styles.icon} source={{ uri: thumbnailPath }} /> : transactionIcon}</View>
-            <View style={styles.transactionContainer}>
-              <View style={styles.transactionRow}>
-                <EdgeText style={styles.transactionText}>{transactionText}</EdgeText>
-                <EdgeText style={isSentTransaction ? styles.negativeCryptoAmount : styles.positiveCryptoAmount}>{cryptoAmountString}</EdgeText>
-              </View>
-              <View style={styles.transactionRow}>
-                <View style={styles.categoryAndTimeContainer}>
-                  {categoryText && <EdgeText style={styles.category}>{categoryText}</EdgeText>}
-                  <EdgeText style={pendingStyle}>{pendingText}</EdgeText>
-                </View>
-                <EdgeText style={styles.fiatAmount}>{fiatAmountString}</EdgeText>
+      <TouchableOpacity onPress={this.handlePress}>
+        <View style={styles.rowContainer}>
+          <View style={[styles.iconContainer, transactionStyle]}>
+            {thumbnailPath ? <Image style={styles.icon} source={{ uri: thumbnailPath }} /> : transactionIcon}
+          </View>
+          <View style={styles.transactionContainer}>
+            <View style={styles.transactionRow}>
+              <EdgeText style={styles.transactionText}>{transactionText}</EdgeText>
+              <EdgeText style={isSentTransaction ? styles.negativeCryptoAmount : styles.positiveCryptoAmount}>{cryptoAmountString}</EdgeText>
+            </View>
+            <View style={styles.transactionRow}>
+              <View style={styles.categoryAndTimeContainer}>
+                {categoryText && <EdgeText style={styles.category}>{categoryText}</EdgeText>}
+                <EdgeText style={pendingStyle}>{pendingText}</EdgeText>
               </View>
+              <EdgeText style={styles.fiatAmount}>{fiatAmountString}</EdgeText>
             </View>
           </View>
-        </TouchableOpacity>
-      </View>
+        </View>
+      </TouchableOpacity>
     )
   }
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
-  container: {
-    flex: 1
-  },
   rowContainer: {
+    flex: 1,
     flexDirection: 'row',
     justifyContent: 'center',
-    paddingVertical: theme.rem(0.25),
-    paddingHorizontal: theme.rem(6 / 16),
-    backgroundColor: theme.tileBackground,
+    paddingVertical: theme.rem(1.5),
+    paddingHorizontal: theme.rem(1.5),
     marginBottom: theme.rem(2 / 16)
   },
   iconContainer: {
     justifyContent: 'center',
     alignItems: 'center',
-    paddingHorizontal: theme.rem(6 / 16)
+    backgroundColor: theme.transactionListIconBackground,
+    paddingHorizontal: theme.rem(0.5),
+    width: theme.rem(3.25),
+    height: theme.rem(3.25),
+    borderWidth: theme.rem(2 / 16),
+    borderRadius: theme.rem(1)
+  },
+  iconSent: {
+    borderColor: theme.negativeText,
+    shadowColor: theme.negativeText,
+    shadowOffset: {
```

Comment:
Change all of the shadow parameters below to use REM values

Reply:
Fixed